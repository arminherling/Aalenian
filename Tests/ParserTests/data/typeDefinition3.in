type Vector2D
{
    x:i32 = 0
    y:i32
}
