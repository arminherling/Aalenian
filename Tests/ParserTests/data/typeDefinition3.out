TypeDefinitionStatement: {
    Name: Vector2D
    Block(2): {
        DeclarationStatement: {
            Field: {
                Name: x
            }
            Type: i32
            Value: {
                Number: 0
            }
        }
        DeclarationStatement: {
            Field: {
                Name: y
            }
            Type: i32
        }
    }
}
